src/plugins/chatview/settings.js

```diff
@@ -0,0 +1,21 @@
+const settings = {
```

Comment:
I'm a little ambivalent about having such small files, just for storing settings, I think an argument could be made that it makes discoverability harder because you have everything spread out into so many files that aren't all visible in your editor.

That said... if we were to go the route of putting settings into separate files, we could also make them JSON files.

Reply:
I am not sure I fully share the concern around the number of files. I think small files with a crystal clear name, that are next to another small set of files with the same characteristics, is not a complex structure to be able to understand at first glance. Also, mentally I much prefer that approach which makes it easier for my brain to quickly capture where is what, and its hierarchy, than having to mentally split different parts of the code to understand it every time I edit one single, bigger file.
The reason why I would put settings in a separate file (not religiously, but at least when they surpass a certain number of lines) is to separate config from functionality. So that when I am working on a certain functionality, and I don't care about the specific settings, I don't have to see them.
Of course this is not an absolute truth, and neither do all plugins have to be structured in the same way.

Reply:
Yeah, it is a relatively common pattern to separate config from functionality.

What do you think about making it a JSON file, to make it even clearer that this is config?